CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,3 @@
+- Add Merge-Patch Support Handling #1283
```

Comment:
```suggestion
- Add NGSI-LD Merge-Patch Support Handling #1283
```

(I think is useful to avoid misunderstand this functionality is related with NGSIv2)